anexia_monitoring/management/commands/anxmonitormodules.py

```diff
@@ -0,0 +1,15 @@
+import asyncio
+import json
+
+from django.core.management.base import BaseCommand, CommandError
+from anexia_monitoring.core import get_python_env_info
+
+
+class Command(BaseCommand):
+    help = "Outputs JSON object describing python runtime and modules. Same as /anxapi/vi/modules"
```

Comment:
Please also add a section in the readme with some minimal docs for the command

Reply:
done